native/word_counter/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "word_counter"
+version = "0.1.0"
+authors = ["pingzing <mcalistern@outlook.com>"]
```

Comment:
how do we want to handle authors, btw? or should I just axe this line entirely?

Reply:
I'd axe that line, yeah. It's part of the Offprint project, so the "authors" are all contributors.

Reply:
If we set up a project email account, it'd be a good idea to put that in here.

Reply:
100% agree.

Reply:
axed :+1: